apps/keyboard/js/layouts/emoji.js

```diff
@@ -403,12 +475,104 @@ Keyboards.emoji = {
       ]
     ]
     }, {     // Page 4 - Symbols
-    panelKeys: [ { compositeKey: '🆗', type: 'emoji' }, { compositeKey: '🔀', type: 'emoji' },
-                 { compositeKey: '🔁', type: 'emoji' }, { compositeKey: '🔂', type: 'emoji' },
-                 { compositeKey: '🆕', type: 'emoji' }, { compositeKey: '🆙', type: 'emoji' },
-                 { compositeKey: '🆒', type: 'emoji' }, { compositeKey: '🆓', type: 'emoji' },
-                 { compositeKey: '🆖', type: 'emoji' }, { compositeKey: '📶', type: 'emoji' },
-                 { compositeKey: '🎦', type: 'emoji' }, { compositeKey: '🈁', type: 'emoji' }
+    panelKeys: [{ compositeKey: '3', type: 'emoji' }, { compositeKey: '0', type: 'emoji' },
```

Comment:
I fail to understand why we need these plain numbers. Even if I try to force these code points shown in Emoji (by append a VS16), I still couldn't see an colored number.

Should we just remove these?